src/components/menu/components/cart/Cart.js

```diff
@@ -0,0 +1,88 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _noop from 'lodash/noop';
+import _sum from 'lodash/sum';
+
+//Components
+import EmptyCart from './components/emptyCart';
+import CartItem from './components/cartItem';
+
+//CSS
+
+import './cart.css';
+
+function calculateTotal(cartItems, quantities) {
+  return cartItems.reduce((total, cartItem) => {
+    const { id, price } = cartItem;
+    return total + price * quantities[id];
+  }, 0);
+}
+
+function renderCartItem(cartItem, quantities, handleEvents) {
+  const { id } = cartItem;
+  return (
+    <CartItem
+      key={id}
+      cartItem={cartItem}
+      quantities={quantities}
+      handleEvents={handleEvents}
+    />
+  );
+}
+
+function renderCartItems(cartItems, quantities, handleEvents) {
```

Comment:
have a component called CartItems, pass the items as a prop and render CartItem there, this function will become redundant there